testdata/filters/stackdriver_outbound.yaml.tmpl

```diff
@@ -18,7 +18,7 @@
           "@type": "type.googleapis.com/google.protobuf.StringValue"
           value: |
             {{- if .Vars.JustSendErrorClientLog }}
-            {"access_logging": "ERRORS_ONLY", "enable_audit_log": true}
+            {{ .Vars.StackdriverFilterClientErrorConfig | fill }}
             {{- else }}
-            {"access_logging": "FULL", "enable_audit_log": true}
-            {{- end }}
+            {{ .Vars.StackdriverFilterClientConfig | fill }}
```

Comment:
To avoid changing existing code, can we add a default else at the end with `{"access_logging": "FULL", "enable_audit_log": true}` and fill in `StackdriverFilterClientConfig` when it is specified?

Reply:
Good idea.. fixed